inductiva/fluids/scenarios/coastal_area/bathymetry.py

```diff
@@ -253,16 +266,43 @@ def plot(
               the matplotlib `Axes` object is returned instead.
         """
 
+        # Create uniform grid for interpolation.
+        x_grid, y_grid = np.meshgrid(np.linspace(*self.x_range, grid_size),
+                                     np.linspace(*self.y_range, grid_size))
+
+        # Interpolate depths to uniform grid.
+        interpolator = scipy.interpolate.LinearNDInterpolator(
+            (self.x, self.y),
+            self.depths,
+        )
+
+        depths_grid = interpolator(x_grid, y_grid)
+
+        # Filter out points that are far from bathymetry locations.
+        tree = scipy.spatial.KDTree(np.c_[self.x, self.y])
+        distance, _ = tree.query(np.c_[x_grid.ravel(), y_grid.ravel()], k=1)
+        distance = distance.reshape(x_grid.shape)
+        depths_grid[distance > max_distance] = np.nan
+
+        # Plot the bathymetry.
+        extent = (
```

Comment:
Extent or ranges?

Reply:
Extent, to follow matplotlib's nomenclature.